### DISCUSSION TITLE
Switching letter's case doesn't work for right Alt

### DISCUSSION THREAD

**Discussion by likern** [helix-editor/helix#10470] [General]
### Summary

From helix tutorial:
```
Type Alt-` to set all selected letters to uppercase.
```
It works for left Alt key, but **doesn't work for right Alt key**

### Helix log

2022-07-11T15:45:59.958 helix_view::editor [ERROR] Failed to initialize the LSP for `source.rust` { cannot find binary path }
2022-07-11T15:50:09.946 helix_view::theme [WARN] Theme: invalid style attribute: modifier
2022-07-11T15:55:29.631 helix_view::editor [ERROR] Failed to initialize the LSP for `source.toml` { cannot find binary path }

### Platform

Linux

### Terminal Emulator

GNOME Terminal 3.42.3 using VTE 0.66.2 +BIDI +GNUTLS +ICU +SYSTEMD

### Helix Version

helix 22.05 (27609f50)

**Comment by archseer** [comment]
Right alt is probably mapped as altGr on your system?

**Comment by rkshthrmsh** [comment]
Facing the same issue. `Flip selection cursor and anchor (<A-;>)` works fine. However, switch to uppercase seems not to be working.

**Comment by Kyzzsa** [comment]
I have the same issue on fedora

**Comment by Kyzzsa** [comment]
https://askubuntu.com/questions/1104092/my-right-alt-key-is-not-working
@likern  @rkshthrmsh 
For everyone facing this issue, try this which immediately solved my problem.
